js/manager.js

```diff
@@ -150,11 +148,18 @@ export const DFPManager = Object.assign(new EventEmitter(), {
 
   registerSlot({ dfpNetworkId, adUnit, sizes, renderOutOfThePage, sizeMapping,
                  targetingArguments, slotId, slotShouldRefresh }) {
-    if (!registeredSlots.hasOwnProperty(slotId)) {
-      registeredSlots[slotId] = { slotId, sizes, renderOutOfThePage,
-                                  dfpNetworkId, adUnit, targetingArguments,
-                                  sizeMapping, slotShouldRefresh, loading: false,
-                                };
+    if (!registeredSlots[slotId]) {
```

Comment:
yeah, please silence the warning if required, we really want to use ``` hasOwnProperty() ```.

btw, eslint is not complaining on my local:

``` ➜  react-dfp git:(master) grunt eslint
Running "eslint:all" (eslint) task

Done, without errors. ```